app/controllers/v1/analysis_responses_controller.rb

```diff
@@ -0,0 +1,37 @@
+class V1::AnalysisResponsesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @response = Response.find_or_create_by(responder: analysis)
+    authorize_action_for @response
+
+    @response.rubric = analysis.rubric
+    if @response.save
+      create_empty_scores(@response)
+      render nothing: true, status: :created
+    else
+      render nothing: true, status: :bad_request
```

Comment:
This is a holdover from existing behavior.  Presently the `ResposnesController` will render nothing on an error.  The other caveat to this is, considering that creating a response can't realistically fail - no validations are run on the model - there's no need in my mind to render errors that won't occur.  The intent here is that if this does change, our tests should cover this.